### ISSUE TITLE
Cannot expand comments

### ISSUE THREAD

**Issue by lucasyvas** [aeharding/voyager#1433]
**Describe the bug**

Cannot expand additional comments hidden by a caret

**To Reproduce**

1. Find any post with comments collapsed behind a caret
2. Try to expand (ex. 1 more reply)
3. A loading icon will display briefly and nothing happens

**Expected behavior**

The comment tree should load and expand

**Screenshots**

![image](https://github.com/aeharding/voyager/assets/3365616/058fac0f-f393-4c5b-a70a-00b6b24619ee)

**Smartphone:**

- Device: iPhone 15
- OS: iOS 17.4.1

**Comment by lucasyvas** [comment]
> Did you block https://lemmy.ca/u/recursive_recursion@programming.dev? That might be why. Nothing that really can be done about that, other that get a fix in Lemmy so that reply count reflects your blocks.

I've never blocked anyone. It happens on all threads.

**Comment by aeharding** [comment]
I'm not sure why you're seeing that then, I cannot reproduce.

**Comment by lucasyvas** [comment]
Do you have any other debugging ideas? It's always been like this for me and I only just finally found it annoying enough to report. I could probably read the code at some point - does it perform an additional network request?

**Comment by aeharding** [comment]
Does it also happen in Lemmy UI when you're logged in there with same account?

**Comment by lucasyvas** [comment]
Actually, that was a good idea. It does not work there either, yet the request does complete in the network panel. Do I have some kind of cosmetic filter on that instance I'm not aware of?

**Comment by lucasyvas** [comment]
The link the parent itself will show the nested comments fine. Found another one here:

https://lemm.ee/comment/11439660

Also attached a screenshot of it in the main comment thread where I click more. And I'll attach the response JSON when clicking that more button.

GET https://lemm.ee/api/v3/comment/list?parent_id=11443652&max_depth=8&limit=999&type_=All

<details>
<summary><b>Response</b></summary>

```json
{"comments":[{"comment":{"id":11443652,"creator_id":2717091,"post_id":30229676,"content":"> Just wait until they’ve helped their 60+ y/o colleagues disable Caps Lock for the 50th time.\n\nYou have no idea how much rage you just unlocked.","removed":false,"published":"2024-04-24T08:41:37.653228Z","deleted":false,"ap_id":"https://lemmy.ca/comment/8783371","local":false,"path":"0.11439660.11443652","distinguished":false,"language_id":0},"creator":{"id":2717091,"name":"nik282000","display_name":"Nik282000","avatar":"https://lemmy.ca/pictrs/image/daa40847-3d53-4a96-9f97-179bc221995f.png","banned":false,"published":"2023-08-22T15:35:12.219537Z","actor_id":"https://lemmy.ca/u/nik282000","bio":" http://nik282000.com/","local":false,"banner":"https://lemmy.ca/pictrs/image/633a0de3-6a3f-4e5b-ae28-618e8a9eaaa3.jpeg","deleted":false,"bot_account":false,"instance_id":50},"post":{"id":30229676,"name":"Young Canadians think retiring at 65 is an outdated concept","url":"https://financialpost.com/fp-work/young-canadians-think-retiring-65-outdated-concept","body":"> Canadians’ views on retirement are shifting dramatically, with the idea of retiring at age 65 being one of the early casualties. Read more.","creator_id":500311,"community_id":377,"removed":false,"locked":false,"published":"2024-04-24T04:16:45.548584Z","deleted":false,"nsfw":false,"embed_title":"Young Canadians think retiring at 65 is an outdated concept","embed_description":"Canadians’ views on retirement are shifting dramatically, with the idea of retiring at age 65 being one of the early casualties. Read more.","thumbnail_url":"https://lemmy.ca/pictrs/image/b275b5c8-129d-4e75-928b-c982bac131a6.jpeg","ap_id":"https://lemmy.ca/post/19931918","local":false,"language_id":0,"featured_community":false,"featured_local":false},"community":{"id":377,"name":"canada","title":"Canada","description":"**What's going on Canada?**\n\n[![](https://img.shields.io/lemmy/canada%40lemmy.ca?style=for-the-badge&amp;logo=lemmy&amp;label=Total%20Subscribers&amp;labelColor=210808&amp;color=531313)](https://lemmy.ca/post/5635462)\n\n---\n---\n\n### Communities\n\n---\n\n::: spoiler 🍁 Meta\n* [Main](https://lemmy.ca/c/main) \n* [Announcements](https://lemmy.ca/c/announcements)\n* [Lemmy.ca Support](https://lemmy.ca/c/lemmy_ca_support) \n:::\n\n---\n\n::: spoiler 🗺️ Provinces / Territories\n* [Alberta](https://lemmy.ca/c/alberta) \n* [British Columbia](https://lemmy.ca/c/britishcolumbia) \n* [Manitoba](https://lemmy.ca/c/manitoba) \n* [New Brunswick](https://lemmy.ca/c/newbrunswick) \n* [Newfoundland and Labrador](https://lemmy.ca/c/newfoundland) \n* [Northwest Territories](https://lemmy.ca/c/nwt) \n* [Nova Scotia](https://lemmy.ca/c/nova_scotia_general) \n* [Nunavut](https://lemmy.ca/c/nunavut) \n* [Ontario](https://lemmy.ca/c/ontario) \n* [Prince Edward Island](https://lemmy.ca/c/pei) \n* [Quebec](https://lemmy.ca/c/quebec) \n* [Saskatchewan](https://lemmy.ca/c/sask) \n* [Yukon](https://lemmy.ca/c/yukon) \n:::\n\n---\n\n::: spoiler 🏙️ Cities\n* [Calgary (AB)](https://lemmy.ca/c/calgary) \n* [Edmonton (AB)](https://lemmy.ca/c/edmonton)\n* [Greater Sudbury (ON)](https://lemmy.ca/c/greater_sudbury) \n* [Halifax (NS)](https://lemmy.ca/c/halifax) \n* [Hamilton (ON)](https://lemmy.ca/c/hamilton) \n* [London (ON)](https://lemmy.ca/c/london)\n* [Mississauga (ON)](https://lemmy.ca/c/mississauga) \n* [Montreal (QC)](https://lemmy.ca/c/montreal) \n* [Nanaimo (BC)](https://lemmy.ca/c/nanaimo)\n* [Oceanside (BC)](https://lemmy.ca/c/oceanside_bc@lemmy.ca)\n* [Ottawa (ON)](https://lemmy.ca/c/ottawa) \n* [Port Alberni (BC)](https://lemmy.ca/c/portalberni)\n* [Thunder Bay (ON)](https://lemmy.ca/c/thunderbay) \n* [Toronto (ON)](https://lemmy.ca/c/toronto) \n* [Vancouver (BC)](https://lemmy.ca/c/vancouver) \n* [Victoria (BC)](https://lemmy.ca/c/victoria) \n* [Waterloo (ON)](https://lemmy.ca/c/waterloo) \n* [Winnipeg (MB)](https://lemmy.ca/c/winnipeg) \n:::\n\n---\n\n::: spoiler 🏒 Sports\nHockey\n- List of All Teams: [Post on /c/hockey](https://lemmy.ca/post/634982)\n- General Community: [/c/Hockey](https://lemmy.ca/c/hockey) \n- [Calgary Flames](https://lemmy.ca/c/calgary_flames) \n- [Edmonton Oilers](https://lemmy.ca/c/edmontonoilers) \n- [Montréal Canadiens](https://lemmy.ca/c/habs@lemmy.world)\n- [Ottawa Senators](https://lemmy.ca/c/ottawasenators)\n- [Toronto Maple Leafs](https://lemmy.ca/c/leaf_nation) \n- [Vancouver Canucks](https://lemmy.ca/c/canucks) \n- [Winnipeg Jets](https://lemmy.world/c/winnipegjets) \n\nFootball (NFL)\n- List of All Teams: `unknown`\n\nFootball (CFL)\n- List of All Teams: `unknown`\n\nBaseball\n- List of All Teams: `unknown`\n- [Toronto Blue Jays](https://lemmy.ca/c/torontobluejays) \n\nBasketball\n- List of All Teams: `unknown`\n- [Toronto Raptors](https://lemmy.ca/c/torontoraptors) \n\nSoccer\n- List of All Teams: `unknown`\n- General Community: [/c/CanadaSoccer](https://lemmy.ca/c/canadasoccer) \n- [Toronto FC](https://lemmy.ca/c/torontofc) \n:::\n\n---\n\n::: spoiler 💻 Universities\n* [UBC](https://lemmy.ca/c/ubc)\n* [UWaterloo](https://lemmy.ca/c/uwaterloo) \n* [SFU](https://lemmy.ca/c/sfu) \n* [UoGuelph](https://lemmy.ca/c/uoguelph)  \n* [OTU](https://lemmy.ca/c/otu)\n* [McGill](https://lemmy.ca/c/mcgill)\n* [U Western](https://lemmy.ca/c/uwo)\n* [UofT](https://lemmy.ca/c/uoft)\n* UofA: `unknown`\n\n:::\n\n---\n\n::: spoiler 👒 Lifestyle\n`coming soon`\n:::\n\n---\n\n::: spoiler 💵 Finance / Shopping\n* [Personal Finance Canada](https://lemmy.ca/c/personalfinancecanada) \n* [BAPCSalesCanada](https://lemmy.ca/c/bapcsalescanada) \n* [Canadian Investor](https://lemmy.ca/c/canadianinvestor) \n* [Buy Canadian](https://lemmy.ca/c/buycanadian) \n* [Quebec Finance](https://lemmy.ca/c/quebecfinance) \n* [Churning Canada](https://lemmy.ca/c/churningcanada) \n:::\n\n---\n\n::: spoiler 🗣️ Politics\n* [Canada Politics](https://lemmy.ca/c/canadapolitics) \n* [NDP](https://lemmy.ca/c/ndp)\n:::\n\n---\n\n::: spoiler 🍁 Other\n* [Canada Politics](https://lemmy.ca/c/canadapolitics) \n* [Ask a Canadian](https://lemmy.ca/c/askacanadian) *___*\n:::\n\n---\n\n### Rules\n\n::: spoiler Reminder that the rules for lemmy.ca also apply here:\n\n1. No bigotry - including racism, sexism, ableism, homophobia, transphobia, or xenophobia.\n2. Be respectful. Everyone should feel welcome here.\n3. No porn.\n4. No Ads / Spamming.\n\n:::","removed":false,"published":"2020-12-14T00:45:17.840800Z","updated":"2024-04-19T07:45:44.342757Z","deleted":false,"nsfw":false,"actor_id":"https://lemmy.ca/c/canada","local":false,"icon":"https://lemmy.ca/pictrs/image/616ced9a-96a1-4b12-96b6-ca84b2fc03b9.mp4","banner":"https://lemmy.ca/pictrs/image/637ecc5b-04fa-4882-b143-7659e090d87d.jpeg","hidden":false,"posting_restricted_to_mods":false,"instance_id":50},"counts":{"comment_id":11443652,"score":8,"upvotes":8,"downvotes":0,"published":"2024-04-24T08:41:37.653228Z","child_count":1},"creator_banned_from_community":false,"creator_is_moderator":false,"creator_is_admin":false,"subscribed":"Pending","saved":false,"creator_blocked":false}]}
```

</details>
<img width="801" alt="Screenshot 2024-04-24 at 12 51 19 PM" src="https://github.com/aeharding/voyager/assets/3365616/83a79601-7557-4491-8de7-0b347c406278">

**Comment by aeharding** [comment]
That request works fine for me unauthenticated. Are you sure there isn't some user or instance block setup on your account, or something like that?

**Comment by lucasyvas** [comment]
It works for me unauthenticated too (auto-expanded even!), but logged in, it's collapsed and doesn't expand. Weird.

This doesn't seem to be an issue with Voyager at least, so I can close this. I tried messing with a lot of settings but it didn't seem to help. The only thing I've noticed is that my subscription to that community is infinitely pending. My language is Undetermined - maybe I can try messing with that.
